docs/en/sql-reference/functions/string-replace-functions.md

```diff
@@ -193,3 +193,33 @@ Result:
 ## translateUTF8
 
 Like [translate](#translate) but assumes `s`, `from` and `to` are UTF-8 encoded strings.
+
+**Syntax**
+
+``` sql
+translateUTF8(s, from, to)
+```
+
+**Parameters**
+
+- `s`: A string type [String](/docs/en/sql-reference/data-types/string.md).
+- `from`: A string type [String](/docs/en/sql-reference/data-types/string.md).
+- `to`: A string type [String](/docs/en/sql-reference/data-types/string.md).
+
+**Returned value**
+
+- `s`: A string type [String](/docs/en/sql-reference/data-types/string.md).
+
+**Examples**
+
+Query:
+
+``` sql
+SELECT translateUTF8('Hello, World!', 'delor', 'DELOR') AS res;
```

Comment:
An example which actually involves non-ASCII UTF-8 strings would be nice.

Maybe

```sql
SELECT translateUTF8('Münchener Straße', 'üß', 'us');

Result: Munchener Strase
```

Reply:
Updated. Thanks for your recommendation.

Reply:
(as a German, I must say that `Munchener Strase` is terrible spelling 😄)